packages/xo-web/src/common/select-objects.js

```diff
@@ -1077,3 +1078,20 @@ export const SelectNetworkConfig = makeSubscriptionSelect(
     }),
   { placeholder: _('selectNetworkConfigs') }
 )
+
+// ===================================================================
+
+export const SelectXoCloudConfig = makeSubscriptionSelect(
+  subscriber =>
+    subscribeCloudXoConfigBackups(configs => {
+      const xoObjects = groupBy(
+        map(configs, config => ({ ...config, type: 'xoConfig' })),
+        'xoa_id'
```

Comment:
Last minute change: all entries are now using camelCase: `xoaId`, `licenseId`, `createdAt`.

My test branch reflects that, please update this PR :pray: